### ISSUE TITLE
More details on the project

### ISSUE THREAD

**Issue by PranavShevkar** [paul-ridgway/LED-Drums-V2#1]
Hello Paul,
I came accross your blog on blockdev and I am very interested in your implementation. I just noticed this repo as an update to your initial solution. Could you please update the README or point me to any documentation for this project? Thanks in advance

**Comment by paul-ridgway** [comment]
Hi,

Thanks for your interest. I've been meaning to write up a blog post for this new version. It's a PlatformIO project and designed to run on a single ESP 32 - the PIN references should be easy enough to find in the code for now?

**Comment by PranavShevkar** [comment]
Thankyou so much for your quick reply. Most of the things are very clear now. Did you use the same arduino midi shield for this project as well?

**Comment by paul-ridgway** [comment]
I actually just ended up making a very simple one with an optoisolator as it was easier and smaller - something like this:

![image](https://github.com/user-attachments/assets/9a67ee42-dc26-42df-9abb-31cb0f763776)

**Comment by PranavShevkar** [comment]
Hi @paul-ridgway,
Thankyou for your explanation and your amazing project. I managed to use a MIDI shield with ESP32 to avoid complexity. Then I modified certain parts of your code to suit my Alesis Nitro Mesh drumset. Because of this project, my drumset looks very cool now. Since I understand the working quite well now, I will write a proper README for others to be able to use this repo.
Thanks.
